package.json

```diff
@@ -282,7 +282,7 @@
     "font-awesome": "4.7.0",
     "fs-extra": "8.1.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "25.5.4",
+    "jest": "^26.6.3",
```

Comment:
@petemill need to check that this doesn't break anything

Reply:
There's at least 1 jest test failure. Taking a look...